container/cypress/e2e/test-app/iframe/iframe-container.cy.js

```diff
@@ -115,4 +115,58 @@ describe('Iframe Container Test', () => {
           });
       });
   });
+
+  it('openAsModal', () => {
+    cy.on('window:confirm', () => false); 
+  
+    cy.get(containerSelector)
+      .shadow()
+      .get('iframe')
+      .then(iframe => {
+        const $body = iframe.contents().find('body');
+        cy.wrap($body)
+          .contains('test openAsModal()')
+          .click();
+  
+        cy.location().should(loc => {
+          expect(loc.href).to.eq('http://localhost:8080/iframe/iframeContainer.html#openAsModal-iframe');
```

Comment:
I think it's safer to check only hash instead of full url - it also relates to test cases below

Reply:
makes sense, thanks :)